### PR TITLE
custom field url open in new_window

### LINKED ISSUE #11300: Open custom URL fields in a new window/tab
### NetBox version

v3.3.9

### Feature type

New functionality

### Proposed functionality

Costum Fields

1. Click Other
2. Click Costum Fields
3. Click Add
4. Add some Mockup Data for Testing. In my Case i select "DCIM > Sides" for example under Models and Name "Test Website".
5. Click Type and select "Type" to URL
6. Click Save
7. Click Organization 
8. Click Site (my Case)
9. Select a random Site in your List and click on it
10. Click on Edit Button  
11. Fill a URL unter Custom Fields like xmas.shellboy.org
12. Click Save
13. Click on the Link now my Site (XMAS greetings surprise for all Netbox Issue reader)  opens in the same windows as netbox 👎 

### Use case

Realy Clear or?

It would be really nice if there would be the possibility that the link also (as with Costum Link) opens in a new window. So that Netbox does not fall out of focus. 

### Database changes

?

### External dependencies

?

### PR DESCRIPTION
**Pull request by ITJamie**
### Fixes: #11300
adds new_window support to the custom_fields URL type.

### CODE REVIEW

**Comment by ITJamie** [comment]
per the convo in the issue topic i made sure that the open-in-new icon is used on the url if the field is set to new_window 

Screenshot 2023-05-29 at 21 51 55

File: netbox/utilities/templates/builtins/customfield_value.html [hunk 1]
```diff
@@ -12,7 +12,11 @@
 {% elif customfield.type == 'datetime' and value %}
   {{ value|annotated_date }}
 {% elif customfield.type == 'url' and value %}
-  <a href="{{ value }}">{{ value|truncatechars:70 }}</a>
+  {% if customfield.new_window %}
+    <a href="{{ value }}" target="_blank">{{ value|truncatechars:70 }} <i class="mdi mdi-open-in-new"></i></a>
```
Comment (ITJamie) [inline netbox/utilities/templates/builtins/customfield_value.html:16]: undecided on if we should add an alt text attribute here.

**Comment by abhi1693** [comment]
Thank you for your interest in contributing to NetBox, however, it appears there is no accepted issue that correlates to this pull request. Before pull requests are opened, we require an accepted issue as per our [contributing guide](https://github.com/netbox-community/netbox/blob/master/CONTRIBUTING.md). Please first open an issue and wait for it to be accepted before further work is done on this pull request.

**Comment by ITJamie** [comment]
https://github.com/netbox-community/netbox/issues/11300 was accepted but timed out

**Comment by ITJamie** [comment]
> https://github.com/netbox-community/netbox/issues/11300 was accepted but timed out

See comment https://github.com/netbox-community/netbox/issues/11300#issuecomment-1377579600

**Comment by abhi1693** [comment]
The comment you have shared does not say the fr is accepted nor it has the correct tags which shows it was ever the case.

**Comment by ITJamie** [comment]
Jeremy reply states that another maintainer could work on this. the community requested this and it was left open for a pr, unfortunately i couldn't get to it before it timed out

**Pull request closed without merging**